### PR TITLE
feat: added marquee animation

### LINKED ISSUE #2160: [feature]: add marquee animation to list of supported dependencies in the landing page
### Is there an existing feature request for this?

- [X] I have searched the existing issues

### Summary

Add marquee effect https://magicui.design/docs/components/marquee to the "Replicate data from any source  " section of keploy landing page. 

### Why should this be worked on?

better design

### Repository

website

### PR DESCRIPTION
**Pull request by Aman172003**
Closes [#2160](https://github.com/keploy/keploy/issues/2160)

https://github.com/user-attachments/assets/1e1de0dd-ff7a-469e-baa3-2107ec70371f

### CODE REVIEW

**Comment by Aman172003** [comment]
@Sonichigo review required!!

**Comment by Aditya-eddy** [comment]
Hey @Aman172003 why did you change the package's version in package.json?

**Comment by Aman172003** [comment]
> Hey @Aman172003 why did you change the package's version in package.json?
> 

I was not able to use the marquee component with current version of eslint dependency. 
It was throwing errors

**Comment by Aman172003** [comment]
@Aditya-eddy is there anything I need to change?

File: components/language.tsx [hunk 1]
```diff
@@ -96,427 +138,19 @@ export default function Language() {
...
-                    width={115}
-                    height={110}
-                    alt="Nutanix Testimonial"
-                  />
-                </span>
-              </div>
-            </div>
+          <div className="relative max-w-6xl px-4 mx-auto overflow-hidden sm:px-6 mt-10 [mask-image:_linear-gradient(to_right,transparent_0,_black_128px,_black_calc(100%-200px),transparent_100%)]">
+            <Marquee pauseOnHover className="[--duration:40s]">
+              {frameworks.map((framework, index) => (
```
Comment (nehagup) [inline components/language.tsx]: It'll be better to keep the 2 unique lists and not repeat frameworks.

File: components/language.tsx [hunk 2]
```diff
@@ -674,7 +323,10 @@ export default function Language() {
             </Link>
 
             {/* Item */}
-            <Link href="https://keploy.io/docs/quickstart/samples-rust/" className="flex items-center justify-center col-span-2 py-2 transition-transform hover:scale-110 md:col-auto">
+            <Link
```
Comment (nehagup) [inline components/language.tsx:330]: there are linting changes added, please remove these.

File: tailwind.config.js [hunk 3]
```diff
@@ -1,220 +1,216 @@
...
-            spacing: {
-                '9/16': '56.25%',
-                '3/4': '75%',
-                '1/1': '100%',
-            },
-            fontFamily: {
-                inter: ['var(--font-inter)', 'sans-serif'],
-                'architects-daughter': ['var(--font-architects-daughter)', 'sans-serif'],
-                pencil: ['Pencil', 'sans-serif'],
+  content: [
```
Comment (nehagup) [inline tailwind.config.js:3]: Please remove linting changes.

**Review by nehagup — CHANGES_REQUESTED** [review]
Thanks for the PR @Aman172003, I've requested some changes.

**Comment by Aman172003** [comment]
Hey @nehagup mam, i have done all the required changes

**Review by nehagup — APPROVED** [review]
This looks good! Thanks @Aman172003 👏

**Pull request merged**